scripts/download_openmeteo_2024_2025.py

```
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+import time
+from datetime import datetime, timedelta
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)